### ISSUE TITLE
acceptance: binary.BigEndian.AppendUint64 undefined

### ISSUE THREAD

**Issue by erikgrinaker** [cockroachdb/cockroach#88221]
I [occasionally](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_EssentialCi/6538931) see this acceptance failure in CI:

```
08:30:10   Run acceptance tests
08:30:10     prep
08:30:14     run
08:30:15       env TZ=America/New_York stdbuf -eL -oL go test -mod=vendor -json -timeout 30m -v -exec ../../build/teamcity-go-test-precompiled.sh ./pkg/acceptance/acceptance.test ./pkg/acceptance -l /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance -b /home/agent/work/.go/src/github.com/cockroachdb/cockroach/cockroach-linux-2.6.32-gnu-amd64
08:30:18       # github.com/cockroachdb/cockroach/pkg/util/uint128
08:30:18       pkg/util/uint128/uint128.go:36:25: binary.BigEndian.AppendUint64 undefined (type binary.bigEndian has no field or method AppendUint64)
08:30:18       pkg/util/uint128/uint128.go:37:26: binary.BigEndian.AppendUint64 undefined (type binary.bigEndian has no field or method AppendUint64)
08:31:15       FAIL  github.com/cockroachdb/cockroach/pkg/acceptance [build failed]
08:31:16       not a single test was parsed, but detected test output: FAIL  github.com/cockroachdb/cockroach/pkg/acceptance [build failed]
08:31:16       exit status 1
```

@miretskiy I see you added this in #87968, can you have a look? Probably because this isn't using Go 1.19.

Jira issue: CRDB-19731

**Comment by rickystewart** [comment]
@healthy-pod is working on an [update](https://github.com/cockroachdb/cockroach/pull/89232/files) to the acceptance tests which I suspect will fix this.

**Comment by rickystewart** [comment]
I think @healthy-pod fixed this with #88221. I'm closing this for now; re-open if I am mistaken.
